### PR TITLE
Fix release notes link

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Comment by thatdocslady** [comment]
@garrett how do we handle updates to haml/yaml files? Is there anything extra we should run/check before we merge? Normally for md files the risk is low, but not sure how to ensure safe merge in this case.

**Comment by garrett** [comment]
You should check it locally, to ensure the formatting looks correct.

However, Travis checks that the site builds. If there's an actual syntax error in a HAML or YAML file, the build will fail and the checks will _not_ pass. Therefore, always make sure all changes to any non-Markdown file go through the pull request process, so that the site won't break*.

_\* And this is the literal form of break. Wonkiness or incorrectness can still show up. There's no way to prevent **that** in a machine automated way. At least the PR + CI process will check for actual breakage. :smiley:_

---

**TL;DR**: Pull requests call Travis CI, which ensures the site builds (but it doesn't check the content).

**Comment by thatdocslady** [comment]
@ekohl plz see above comment, can you confirm that you ran this fix locally and that it works? Unfortunately we don't have pre-merge staging to view the site output yet...

**Comment by ekohl** [comment]
@thatdocslady I have now, but I simply used the edit in github as linked at the bottom of the page. I recommend you add a CONTRIBUTING.md to the repository so users who submit a pull request know they should manually verify it.

**Comment by thatdocslady** [comment]
@ekohl thanks for the suggestion, will add contribution guidelines link :)

**Comment by thatdocslady** [comment]
@ekohl can you please check if the new contribution guidelines file appears when you try to open a new pull request? I can't seem to find it and I have a feeling it's because I have commit rights ;)

**Comment by ekohl** [comment]
@thatdocslady it does. Even though you have commit rights, you should be able to open a PR.
pr-screenshot
